docs/business/alternate_domains.rst

```diff
@@ -0,0 +1,32 @@
+Alternate Domains
+=================
+
+Read the Docs supports a number of custom domains for your convenience.
+
+Subdomain support
+-----------------
+
+Every project has a subdomain that is available to serve its documentation.
+If you go to ``<organization-slug-project-slug>.readthedocs-hosted.com``,
+it should show you the latest version of documentation.
```

Comment:
I'm -1 on having this level of detail in both the commercial hosting docs and the community docs. I don't think it helps to describe slugs or our slug pattern. I think what we are trying to communicate is:

* By default, your project is hosted from a subdomain on one of our domains
* If you need a custom domain, continue on to custom domain setup